dm/master/server_test.go

```diff
@@ -175,8 +175,16 @@ func TestMaster(t *testing.T) {
 		t.Fatal(err)
 	}
 	// inject *testing.T to testMaster
-	s := testSuite.(*testMaster)
-	s.testT = t
+	s, ok := testSuite.(*testMaster)
+	if ok {
```

Comment:
How can this type cast fail, I think this check is redundant

Reply:
in L184, `s` possibly be `openAPISuite`

Reply:
No. But the `testSuite` and `ts` are never mistaken with each other.

Reply:
😬 yes, my bad , fixed in https://github.com/pingcap/dm/pull/1999/commits/a6935915d18e82d776e4cad867cc8bca62ef14a1